packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/ListView/tests/index.test.js

```diff
@@ -34,1310 +53,145 @@ describe('Admin | containers | ListView', () => {
-                    </table>
-                  </div>
-                </div>
-                <div>
-                  <hr
-                    class="c47 c48"
-                  />
-                  <button
-                    class="c49 c50"
-                  >
-                    <div
-                      class="c7"
-                    >
-                      <div
-                        aria-hidden="true"
-                        class="c51 c52"
-                      >
-                        <svg
-                          fill="none"
-                          height="1rem"
-                          viewBox="0 0 24 24"
-                          width="1rem"
-                          xmlns="http://www.w3.org/2000/svg"
-                        >
-                          <path
-                            d="M24 13.604a.3.3 0 0 1-.3.3h-9.795V23.7a.3.3 0 0 1-.3.3h-3.21a.3.3 0 0 1-.3-.3v-9.795H.3a.3.3 0 0 1-.3-.3v-3.21a.3.3 0 0 1 .3-.3h9.795V.3a.3.3 0 0 1 .3-.3h3.21a.3.3 0 0 1 .3.3v9.795H23.7a.3.3 0 0 1 .3.3v3.21Z"
-                            fill="#212134"
-                          />
-                        </svg>
-                      </div>
-                      <div
-                        class="c53"
-                      >
-                        <span
-                          class="c54"
-                        >
-                          Create new webhook
-                        </span>
-                      </div>
-                    </div>
-                  </button>
-                </div>
-              </div>
-            </div>
-          </main>
-        </div>
-      </div>
-    `);
+    expect(getByText('Are you sure you want to delete this?')).toBeInTheDocument();
   });
 
-  describe('Shows a loading state', () => {
-    it('should show a loader when it is loading for the permissions', () => {
-      useRBAC.mockImplementation(() => ({
-        isLoading: true,
-        allowedActions: { canUpdate: true, canCreate: true, canRead: true, canDelete: true },
-      }));
+  it('should delete all webhooks', async () => {
+    const user = userEvent.setup();
 
-      render(App);
+    useRBAC.mockImplementation(() => ({
+      isLoading: false,
+      allowedActions: { canUpdate: true, canCreate: true, canDelete: true },
+    }));
 
-      expect(screen.getByTestId('loader')).toBeInTheDocument();
+    useNotification.mockImplementation(() => jest.fn());
+
+    const { container, getByText, getByRole } = render(App);
+    await waitFor(() => {
+      screen.getByText('http:://strapi.io');
     });
 
-    it('should show a loader when it is loading for the data and not for the permissions', () => {
-      useRBAC.mockImplementation(() => ({
-        isLoading: false,
-        allowedActions: { canUpdate: true, canCreate: true, canRead: true, canDelete: true },
-      }));
+    fireEvent.click(container.querySelector('#select-all'));
+    fireEvent.click(container.querySelector('#delete-selected'));
```

Comment:
same here, don't use querySelector, it's an absolute last resort after `testid`.